Sources/PointFree/Account/Cancel.swift

```diff
@@ -187,7 +187,7 @@ private let formRowView = View<Stripe.Subscription> { subscription in
       form([action(path(to: .cancel)), method(.post)], [
         button(
           [`class`([Class.pf.components.button(color: .red), Class.margin([.mobile: [.top: 3]])])],
-          ["Cancel My Subscription"])
+          ["Cancel my subscription"])
```

Comment:
Not sure what we want our capitalization standard to be for actions but wanted to be consistent. Title-case or first-word?

Reply:
I vote first word!